.changeset/twelve-ducks-mix.md

```diff
@@ -0,0 +1,6 @@
+---
+'@keystonejs/app-admin-ui': patch
```

Comment:
you can merge changesets I guess.
manually editing is fine.

Reply:
Well, I wanted these to appear as patch changes.

Reply:
you need to split PR if you want separate patch changes independent of this change, having minor and extra patch change, what value it has